src/pages/MyNotes/components/NotesList/NotesList.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from 'react';
+import { ListItem } from '@mui/material';
+
+import { Note } from 'pages/MyNotes';
+import { INotesList } from 'pages/MyNotes/types';
+
+import { StyledList, StyledListItemButton } from './styles';
+
+const NotesList: FC<INotesList> = ({ notes, ...props }) => (
+  <StyledList>
+    {notes?.map((page) =>
+      page.map((note) => (
+        <ListItem disablePadding key={note?.id}>
+          <StyledListItemButton $isActive={props.activeNote?.id === note.id}>
```

Comment:
```suggestion
          <StyledListItemButton $isActive={activeNote?.id === note.id}>
```

Reply:
also add constant before return statement like const isNoteActive = activeNote?.id === note.id and reuse it here

Reply:
why did you use $ here?